src/core_plugins/elasticsearch/lib/kibana_index_mappings.js

```diff
@@ -1,16 +1,24 @@
-export const mappings = {
+const currentMappings = {
```

Comment:
The server is an instance of a class, and all of the plugins/modules that get mutated need to bind there state to that class so that multiple instances of the server don't collide/share state accidentally. This isn't something we're doing in kibana (yet) but we do it in the tests all the time. Most of the time those tests ignore all the plugins because of issues like this, but it's definitely a best practice that could/should make it into the style guide.